src/lib/y2storage/dialogs/guided_setup.rb

```diff
@@ -53,19 +53,39 @@ def dialog_title
       def dialog_content
         MarginBox(
           2, 1,
-          RadioButtonGroup(
-            Id(:mode),
-            VBox(
-              Left(RadioButton(Id(:mode_partition), _("Partition-based"), partition_selected?)),
-              VSpacing(1),
-              Left(RadioButton(Id(:mode_lvm), _("LVM-based"), lvm_selected?)),
-              VSpacing(1),
-              Left(RadioButton(Id(:mode_encrypted), _("Encrypted LVM-based"), encrypted_selected?))
+          VBox(
+            RadioButtonGroup(
+              Id(:mode),
+              VBox(
+                Left(RadioButton(Id(:mode_partition), _("Partition-based"), partition_selected?)),
+                VSpacing(1),
+                Left(RadioButton(Id(:mode_lvm), _("LVM-based"), lvm_selected?)),
+                VSpacing(1),
+                Left(RadioButton(Id(:mode_encrypted), _("Encrypted LVM-based"), encrypted_selected?))
+              )
+            ),
+            VSpacing(0.2),
+            Left(
+              HBox(
+                HSpacing(8),
+                Password(Id(:encryption_password), _("Enter encryption password"))
```

Comment:
This field should be there twice to make sure the user typed the same thing consistently; since the widget is in password mode, the user can't see what he types. One typo, and he will never be able to decrypt that LVM again.